avm/res/fabric/capacity/main.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.29.47.4906",
```

Comment:
Not mandatory but just a heads up that a newer version of the bicep cli is available